app.js

```diff
@@ -66,14 +86,18 @@
     },
 
     renderSearchForm: function(searchType) {
-      var searchFields = this.renderTemplate('search-form-' + searchType);
+      var searchFields = this.renderTemplate('search-form-' + searchType, {locales: this.locales});
       var newItemPath = NEW_ITEM_PATH[searchType];
-      var isDcSearch = searchType == 'dynamicContent';
+      var searchFormType = {};
+      searchFormType[searchType] = true;
```

Comment:
I did it this way because you can't evaluate equivalence in handlebars templates, only T/F.